pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -39,22 +40,46 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) *apps.Deployment {
 			Labels: c.getPodLabels(mgrConfig.DaemonID),
 		},
 		Spec: v1.PodSpec{
-			InitContainers: []v1.Container{
-				c.makeSetServerAddrInitContainer(mgrConfig, "dashboard"),
-				c.makeSetServerAddrInitContainer(mgrConfig, "prometheus"),
-			},
+			InitContainers: []v1.Container{},
 			Containers: []v1.Container{
 				c.makeMgrDaemonContainer(mgrConfig),
 			},
 			ServiceAccountName: serviceAccountName,
 			RestartPolicy:      v1.RestartPolicyAlways,
-			Volumes: append(
-				opspec.DaemonVolumes(mgrConfig.DataPathMap, mgrConfig.ResourceName),
-				keyring.Volume().Admin(), // ceph config set commands want admin keyring
-			),
-			HostNetwork: c.HostNetwork,
+			Volumes:            opspec.DaemonVolumes(mgrConfig.DataPathMap, mgrConfig.ResourceName),
+			HostNetwork:        c.HostNetwork,
 		},
 	}
+
+	needHttpBindFix := true
+
+	// if luminous and >= 12.2.12
+	if c.clusterInfo.CephVersion.IsLuminous() &&
```

Comment:
Can't we simply do `c.clusterInfo.CephVersion.IsAtLeast(cephver.CephVersion{Major: 12, Minor: 2, Extra: 12})`, this would imply `IsLuminous` right?

Reply:
IsAtLeast compares across major versions. So, 13.2.5 is at least 12.2.12, but 13.2.5 doesn't have the fix. Alternatively we could add a Major-constrained IsAtLeast variant.

Reply:
I see, thanks!